### ISSUE TITLE
[jorg] HTML-Code shown in Team Reports at Startpage

### ISSUE THREAD

**Issue by sandramay0905** [joomla/joomla-websites#1611]
As title say:

> ![grafik](https://user-images.githubusercontent.com/78906980/114711492-4f09d400-9d2f-11eb-8161-8d5d46a36026.png)

**Comment by hvdmeer** [comment]
I'm not seeing any code, do you still see it @sandramay0905 ?

**Comment by sandramay0905** [comment]
Its only shown at Startpage of joomla.org below "Team Reports", not at Article itself.

Now there is a new Article at "Team Reports" so the Bug Squad-Report and his HTML-Code isn't shown, i close my report.

I suggest to inform the author as i have seen the HTML-Code in his reports below "Team Reports" in the past which looks not good for Joomla!.

**Comment by sandramay0905** [comment]
Startpage | Article view
---- | ----
![Bildschirmfoto 2021-04-18 um 06 08 47](https://user-images.githubusercontent.com/78906980/115133999-dcd60f80-a00c-11eb-809b-ab136573081e.png) | ![Bildschirmfoto 2021-04-18 um 06 08 29](https://user-images.githubusercontent.com/78906980/115134005-eb242b80-a00c-11eb-9387-e760d788a82c.png)

**Comment by ot2sen** [comment]
See it in the announcements feeds module at homepage of j.org
![18-04-2021_jorg_homepage_announcm_feeds_module](https://user-images.githubusercontent.com/790445/115134542-61c32800-a011-11eb-96e3-7f3282f69442.png)

**Comment by conconnl** [comment]
I have analyzed it and it seems to be an issue on how Certain people create reports on the Volunteers portal.
It looks like this report is coming from Word or something like that, I pressed the "Clean Formatting" button in the report and now it's displaying normally.
Somehow the Feed module can't process it correctly, I have no clue why and I'm not a programmer to check any related code.

@sanderpotjer any idea how we can make sure that reports created on the JVP, doesn't make the Main site show html code?

[feed.txt](https://github.com/joomla/joomla-websites/files/6330738/feed.txt)
